cmd/create/create.go

```diff
@@ -0,0 +1,22 @@
+package create
+
+import (
+	"realcloud.tech/pligos/pkg/helm"
+	"realcloud.tech/pligos/pkg/pligos"
+)
+
+type Config struct {
+	ConfigPath  string
+	ContextName string
+	ChartPath   string
+}
+
+func Create(config Config) error {
```

Comment:
`cmd/compile/compile.go` and `cmd/create/create.go` look very different, and this fact introduces different code styles/standards already in this PR.

E.g.:

1. There's a pligos.CreateConfig but no pligos.CompileConfig, or any other pligos.CmdConfig.
2. Each command has a Config struct, but Create has struct Config and pligos.CreateConfig, why multiple configs? What's the meaning of this?
3. pligos.Compiler is a rich struct (has multiple attributes) and a method New(), helm.Creator has none, and you instantiate it without New() as in ln 20
4. The download command need to be observed as well, it follows the same problem, the code style doesn't seem to follow one logic. Maybe everything that was before downloader.Download should be encapsulated in a method. At least this seems to be the solution in compile and create.